src/Serialization/upgrades/0.11.3.jl

```diff
@@ -0,0 +1,120 @@
+################################################################################
+# Upgrade Summary
+# Types of the form `Vector{T}` are only serialized with backref when
+# `is_basic_serialization_type(T) == false`. If
+# `is_basic_serialization_type(T) == true` the type `Vector{T}` does not
+# serialize `T` in the entries, but instead as a part of the `Vector` serialization.
+# Instances of the form `Tuple{T}` with `is_basic_serialization_type(T) == true` 
+# are no longer serialized with backref, however these types were already storing
+# there types outside of the JSON array of entries
+
+push!(upgrade_scripts, UpgradeScript(
+    v"0.11.3", # version this script upgrades to
+    function (s::DeserializerState, dict::Dict)
+        # file comes from polymake
+        if haskey(dict, :_ns)
+            haskey(dict[:_ns], :polymake) && return dict
+        end
+
+        # moves down tree to point where type exists in dict
+        # since we are only doing updates based on certain types
+        if !haskey(dict, :type)
+            upgraded_dict = Dict{Symbol, Any}()
+            for (key, value) in dict
+                if value isa String || value isa Vector{String} || key == :field_types
+                    upgraded_dict[key] = value
+                else
+                    if typeof(value) <: Dict{Symbol, Any}
```

Comment:
```suggestion
                    if value isa Dict{Symbol, Any}
```
Also, maybe turn the `else` plus `if` into an `elseif` (and then adjust indentation accordingly) ?